typescript/test/test_application_js/integration-test.js

```diff
@@ -38,7 +38,7 @@ describe('IntegrationTest for showcase library', () => {
       // Download server
       process.chdir(SHOWCASE_SERVER);
       try {
-        const command = `curl -L https://github.com/googleapis/gapic-showcase/releases/download/v${GAPIC_SHOWCASE_VERSION}/gapic-showcase-${GAPIC_SHOWCASE_VERSION}-${OS}-amd64.tar.gz > gapic-showcase-server.tar.gz`;
+        const command = `curl -L https://github.com/googleapis/gapic-showcase/releases/download/v${GAPIC_SHOWCASE_VERSION}/gapic-showcase-${GAPIC_SHOWCASE_VERSION}-${OS}-${ARCH}.tar.gz > gapic-showcase-server.tar.gz`;
```

Comment:
```
const ARCH = 'amd' + process.arch.toString().substring(1);
```
This is wrong (won't work for other architectures, e.g. arm or x86). Better to do `const ARCH = process.arch == "x64" ? "amd64" : process.arch;` or something like that.  (can be made in a separate PR that will cover all the places where this code exists)

Reply:
Got it! so the value `process.arch`might be: 'arm', 'arm64', 'ia32', 'mips','mipsel', 'ppc', 'ppc64', 's390', 's390x', 'x32', and 'x64'
and based on release page of gapic-showcase
gapic-showcase-0.6.1-darwin-amd64.tar.gz
gapic-showcase-0.6.1-linux-amd64.tar.gz
gapic-showcase-0.6.1-linux-arm.tar.gz
gapic-showcase-0.6.1-protos.tar.gz
gapic-showcase-0.6.1-windows-amd64.tar.gz
gapic-showcase-0.6.1.desc
I think you are right, `const ARCH = process.arch == "x64" ? "amd64" `  should work, thanks!